util/metrics/prometheus.py

```diff
@@ -60,10 +60,10 @@
 )
 gc_iterations = Counter("quay_gc_iterations", "number of iterations by the GCWorker")
 
-secscan_index_request_duration = Histogram(
-    "quay_secscan_index_duration_seconds",
+secscan_request_duration = Histogram(
+    "quay_secscan_request_duration_seconds",
     "seconds taken to make an index request to the secscan service",
-    labelnames=["status"],
+    labelnames=["method", "route", "status"],
```

Comment:
`Route` seems like a bad name, as it seems to be `action` at the call site and I initially thought it would be recording the path used in the request.

Reply:
Changed it from "route" to "action".